### DISCUSSION TITLE
404 error github.io page

### DISCUSSION THREAD

**Discussion by angeella** [community/community#119770] [Other Feature Feedback, Questions, & Ideas]
### Select Topic Area

Bug

### Body

My personal webpage, available at angeella.github.io, has not been working for a few days. I didn't make any major modifications recently.
<img width="542" alt="Screenshot 2024-04-16 111218" src="https://github.com/community/community/assets/36375295/d0a3924a-3e9a-41c8-9427-a701e28c2413">

**Comment by markolalovic** [comment]
Navigate to your personal web site repository. Under your repository name, click  on "Settings" tab:

![Settings tab](https://docs.github.com/assets/cb-28266/mw-1440/images/help/repository/repo-actions-settings.webp)

Then in the sidebar, click on "Pages". If your website is up and running, you should see:

![Pages](https://docs.github.com/assets/cb-82936/mw-1440/images/help/pages/click-private-pages-url-to-preview.webp)

Otherwise, on "Pages" under "Build and deployment", check that the appropriate branch is selected as the publishing source. 

Refer to the documentation: [Configuring a publishing source for your GitHub Pages site](https://docs.github.com/en/enterprise-cloud@latest/pages/getting-started-with-github-pages/configuring-a-publishing-source-for-your-github-pages-site) for step-by-step instructions.

**Comment by WHP-Fernando** [comment]
try in another web browser
